libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -636,6 +656,13 @@ private async Task<DialogTurnResult> CheckForMultiTurnPromptAsync(WaterfallStepC
             return await stepContext.NextAsync(stepContext.Result, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Gets the options the dialog will use to display query results to the user.
+        /// </summary>
+        /// <param name="stepContext">The <see cref="WaterfallStepContext"/> for the current turn of conversation.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> for the current turn of conversation.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        /// <remarks>If the task is successful, the result contains the response options to use.</remarks>
```

Comment:
Consider improving the remarks text. The comment should have a link to the reponseOptions which are actually called DialogTurnResults.

Reply:
Added Reference link.

---
In reply to: [430551097](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430551097) [](ancestors = 430551097)